gegelatilib/include/log/LABasicLogger.h

```diff
@@ -0,0 +1,70 @@
+#ifndef GEGELATI_LABASICLOGGER_H
+#define GEGELATI_LABASICLOGGER_H
```

Comment:
The unwritten convention so far for guards is to use 
`#ifndef FILE_NAME_H`
In your case
`#ifndef LA_BASIC_LOGGER_H`